opba-banking-rest-impl/src/main/java/de/adorsys/opba/tppbankingapi/controller/TppBankSearchController.java

```diff
@@ -51,7 +52,8 @@
         if (onlyActive == null) {
             onlyActive = true;
         }
-        List<BankDescriptor> banks = bankService.getBanks(keyword, start, max, onlyActive);
+
+        List<BankDescriptor> banks = keyword.isBlank() ? Collections.emptyList() : bankService.getBanks(keyword, start, max, onlyActive);
```

Comment:
We should return in response.keyword actual value the client has used - if it was empty string - empty string and not null

Reply:
done